frontend/src/pages/TaskOverviewPage.tsx

```diff
@@ -33,15 +33,33 @@ import { MissingRatings } from '../components/MissingRatings';
 import { TaskModalButtons } from '../components/TaskModalButtons';
 import { apiV2 } from '../api/api';
 import { LoadingSpinner } from '../components/LoadingSpinner';
+import { EditableTextWithButtons } from '../components/EditableText';
 
 const classes = classNames.bind(css);
 
 export const getTaskOverviewData = (
   task: Task,
   editable: boolean,
   users: RoadmapUser[],
+  roadmapId: number,
 ) => {
   const { value, complexity } = valueAndComplexitySummary(task);
+  const [patchTaskTrigger] = apiV2.usePatchTaskMutation();
```

Comment:
This is a hook, so it can't be used like this in a function. Furthermore, this is called conditionally, so there are some errors from different hooks at different renders.